.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,16 +5,19 @@
 - [ ] The top-level JSON objects are sorted alphabetically
 - [ ] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update
 
-#### for websites-with-shared-credential-backends.json
```

Comment:
I think we should also make an update to the top-level `README.md` file so contributors won't continue to be update `websites-with-shared-credential-backends.json`. We can remove mentions of this file except maybe calling out it's deprecated and generated for legacy reasons in favor of these new files you've added.